### PR TITLE
Add aria-required attribute

### PR DESCRIPTION
**Pull request by SteveEdson**

### CODE REVIEW

**Review by benoitgrelard — APPROVED** [review]
Thanks for the contribution!
Would you be able to run `yarn version check --interactive` and set it as a `patch`.
You may also need to run `yarn version check --interactive` a second time to set the overall project as `decline`.
Then you can commit the version file which should clear the CI checks.

🙏

**Comment by SteveEdson** [comment]
@benoitgrelard no problem, updated. Thanks

File: .yarn/versions/4a91906d.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 releases:
-  "@radix-ui/react-tabs": patch
+  "@radix-ui/react-radio-group": patch
 
 declined:
   - primitives
+  - "@radix-ui/react-tabs"
```
Comment (benoitgrelard) [inline .yarn/versions/4a91906d.yml:6]: I'm confused why `tabs` is here? Did it show up in the list to set versions when you ran the command? 🤔

Comment (benoitgrelard) [reply .yarn/versions/4a91906d.yml:6]: On closer inspection, it looks like you have modified an existing version file. Do you know what happened?

Comment (SteveEdson) [reply .yarn/versions/4a91906d.yml:6]: Not sure to be honest! I just ran the version command and it came up, so I set it to decline

Comment (benoitgrelard) [reply .yarn/versions/4a91906d.yml:6]: hmmm, that's really odd, we'll take a look.

**Comment by benoitgrelard** [comment]
Hey I raised a separate PR to fix the version issue, but thanks for your contribution!
You'll see in the description I did change one thing.
I'll close your PR in favour of #1422 now.

**Pull request closed without merging**